docker/ubuntu-2204-nightly/Dockerfile

```diff
@@ -0,0 +1,16 @@
+FROM ubuntu:22.04
```

Comment:
Hi @khalyomede I'm one of the people @Anton-4 mentioned in the PR who talked about Docker on zulip. For my experiments with a dedicated Roc docker image I tried Debian bullseye - it worked. I have two questions:

- if Ubuntu is built on Debian, would it make sense to go directly for Debian? Or maybe even Alpine to keep things small.
- would it make sense to use `@latest` tag? the benefit of this approach is that we'd get quick feedback. The tests would fail indicating that the new OS changes are incompatible with Roc

Keep in mind I'm a Sunday docker user, I might be wrong so post this feedback as questions.

Reply:
Hi Jakub,

As per my discussion in the related issue and my tests, using Roc on Alpine is not trivial and would probably require some research on what dynamic libraries are needed in order for Roc to work properly on Alpine.

And to address your second point, using the `latest` tag would be problematic for reproducability since it is the sole point of Docker and most IAC solutions out there.

If you need to tests things out, you can use a combination of dynamic image name and a GitHub Action Matrix in order to test against multiple versions of Debian if you'd like.

```dockerfile
ARG DEBIAN_VERSION=12.0.0 #hypothetic Debian version, must refer to the available tags

FROM debian:$DEBIAN_VERSION
```

Reply:
> if Ubuntu is built on Debian, would it make sense to go directly for Debian? Or maybe even Alpine to keep things small.

I do not have a strong opinion on one or the other, what you say makes sense, maybe we can support Ubuntu in a first time, if we have some request we may create another tag without any issues (maintaining one or two will not cause a lot of resources since the OSes are quite close to each other).